va/va.go

```diff
@@ -649,13 +705,20 @@ func detailedError(err error) *probs.ProblemDetails {
 		} else if syscallErr, ok := netErr.Err.(*os.SyscallError); ok &&
 			syscallErr.Err == syscall.ECONNREFUSED {
 			return probs.ConnectionFailure("Connection refused")
+		} else if syscallErr, ok := netErr.Err.(*os.SyscallError); ok &&
+			syscallErr.Err == syscall.ENETUNREACH {
+			return probs.ConnectionFailure("Network unreachable")
```

Comment:
This seems like the wrong problem type, a network unreachable error is almost definitely indicating something wrong with our network stack and should trigger 500s we want to alert on.

Reply:
We can't trigger 500's from the VA, since it's async. And I'd like to be able to return this to the user, both because it's handy for our test, and because it can indicate something wrong. For instance, they may provide an actual unreachable IP address that is not covered by the private-address blocks in our DNS code. That doesn't necessarily mean something is wrong with our stack.